html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1> Coding Form!</h1>
+    <form>
+      <label for="email">
+        Email
+        <input type="text" name="email"/><br>
+      </label>
```

Comment:
If the `input` is inside the `label`, the `for` attribute isn't needed. It's also not doing anything, because it looks for an `id`. So you would need to give the `input` an `id` of "email".